app/views/layouts/application.html.erb

```diff
@@ -45,7 +45,7 @@
           <nav class="header__nav-links">
             <% if signed_in? %>
               <a href="<%= profile_path(current_user.display_id) %>" class="header__nav-link mobile__header__nav-link">
-                <%= truncate current_user.name %>
+                <span><%= truncate current_user.name %></span>
```

Comment:
I think that it does'nt need to truncate user name. Because if size of user name overflows width of display, it would be ellipsed by [CSS](https://github.com/rubygems/rubygems.org/pull/1642/files#diff-6f6b88a6b5d4a1e4e8424b58b959c29aR157). But I thought that I should concentrate bugfix at this pull request, so I didn't change logic for truncate username or not.